src/darwin/CHIPTool/CHIPTool/View Controllers/QRCode/QRCodeViewController.m

```diff
@@ -589,12 +589,13 @@ - (void)parseOptionalData:(CHIPSetupPayload *)payload
 - (void)handleRendezVous:(CHIPSetupPayload *)payload
 {
     switch (payload.rendezvousInformation) {
-    case kRendezvousInformationThread:
-    case kRendezvousInformationEthernet:
+    case kRendendezvousInformationOnNetwork:
+    case kRendendezvousInformationSoftAPOnNetwork:
+    case kRendendezvousInformationBLEOnNetwork:
```

Comment:
if `kRendendezvousInformationBLEOnNetwork` and the commissioner supports discovery over BLE, it should be used instead of returning an error. Same for Soft AP.